task/01-strings-tasks.js

```diff
@@ -201,7 +210,22 @@ function extractEmails(str) {
  *
  */
 function getRectangleString(width, height) {
-    throw new Error('Not implemented');
+	const symbol1 = '─';
```

Comment:
figure out better names for variables. With these names you might as well just use string literals